_shared/project/tox.ini

```diff
@@ -131,7 +134,7 @@ commands =
     lint: pycodestyle src tests bin
 {% endif %}
     tests: coverage run -m pytest --failed-first --new-first --no-header --quiet {posargs:tests/unit/}
-    functests: pytest --failed-first --new-first --no-header --quiet {posargs:tests/functional/}
+    functests: python -m pytest --failed-first --new-first --no-header --quiet {posargs:tests/functional/}
```

Comment:
For whatever reason when you just run `pytest` directly it crashes trying to find the cookiecutter pytest plugins, but if you run it with `python -m` (which is the recommended way to run things anyway) it works